server/daos/course_student.js

```diff
@@ -0,0 +1,62 @@
+// import database
+// import modules
+
+const db = require('../db/db.js');
+
+// it creates the course_student table
+exports.createCourse_StudentTable = function() {
+    return new Promise ((resolve,reject) => {
+        const sql = `CREATE TABLE Course_Student (course_id TEXT NOT NULL, student_id TEXT NOT NULL,
```

Comment:
both course_id and student_id should be integers

Reply:
got it